elastic/beats
# Add auditd example with Auditbeat in kubernetes manifests

Add an example configuration of the auditd module in the Auditbeat
reference manifest, including the processors needed for enrichement of
events.

For enrichement it makes use of #15947, included in 7.7.

### How to test?

* Run Auditbeat in kubernetes with the reference configuration.
* Exec a command inside a container, check that an event about this is collected.
* Try to access or write to a file without permissions in a container, check that an event about this is collected.
* Check that events collected about containers include the kubernetes metadata.
